File: vertx-lang-php-jphp/src/main/php/io/vertx/php/core/http/HttpClientRequest.php
<?php 
namespace io\vertx\php\core\http;
use io\vertx\php\core\MultiMap;
use io\vertx\php\core\buffer\Buffer;
class HttpClientRequest
{
    
    private function __construct()
    {

    }
    /**
     * ()
     * @return string
     * methodSize : 1
     */
    public function absoluteURI(/*0*/)
    {

    }
    /**
     * ()
     * @return HttpConnection
     * methodSize : 1
     */
    public function connection(/*0*/)
    {

    }
    /**
     * (callable handler)
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function connectionHandler(/*1*/$arg0)
    {

    }
    /**
     * (callable handler)
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function continueHandler(/*1*/$arg0)
    {

    }
    /**
     * (callable handler)
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function drainHandler(/*1*/$arg0)
    {

    }
    /**
     * (string chunk)
     * @return Void
     * (Buffer chunk)
     * @return Void
     * (string chunk, string enc)
     * @return Void
     * ()
     * @return Void
     * methodSize : 4
     */
    public function end(/*2*/$arg0, $arg1)
    {

    }
    /**
     * (callable endHandler)
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function endHandler(/*1*/$arg0)
    {

    }
    /**
     * (callable handler)
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function exceptionHandler(/*1*/$arg0)
    {

    }
    /**
     * ()
     * @return string
     * methodSize : 1
     */
    public function getHost(/*0*/)
    {

    }
    /**
     * ()
     * @return string
     * methodSize : 1
     */
    public function getRawMethod(/*0*/)
    {

    }
    /**
     * (callable handler)
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function handler(/*1*/$arg0)
    {

    }
    /**
     * ()
     * @return MultiMap
     * methodSize : 1
     */
    public function headers(/*0*/)
    {

    }
    /**
     * ()
     * @return bool
     * methodSize : 1
     */
    public function isChunked(/*0*/)
    {

    }
    /**
     * ()
     * @return enum
     * methodSize : 1
     */
    public function method(/*0*/)
    {

    }
    /**
     * ()
     * @return string
     * methodSize : 1
     */
    public function path(/*0*/)
    {

    }
    /**
     * ()
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function pause(/*0*/)
    {

    }
    /**
     * (callable handler)
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function pushHandler(/*1*/$arg0)
    {

    }
    /**
     * (string name, string value)
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function putHeader(/*2*/$arg0, $arg1)
    {

    }
    /**
     * ()
     * @return string
     * methodSize : 1
     */
    public function query(/*0*/)
    {

    }
    /**
     * ()
     * @return bool
     * (num code)
     * @return bool
     * methodSize : 2
     */
    public function reset(/*1*/$arg0)
    {

    }
    /**
     * ()
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function resume(/*0*/)
    {

    }
    /**
     * ()
     * @return HttpClientRequest
     * (callable completionHandler)
     * @return HttpClientRequest
     * methodSize : 2
     */
    public function sendHead(/*1*/$arg0)
    {

    }
    /**
     * (bool chunked)
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function setChunked(/*1*/$arg0)
    {

    }
    /**
     * (bool followRedirects)
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function setFollowRedirects(/*1*/$arg0)
    {

    }
    /**
     * (string host)
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function setHost(/*1*/$arg0)
    {

    }
    /**
     * (string method)
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function setRawMethod(/*1*/$arg0)
    {

    }
    /**
     * (num timeoutMs)
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function setTimeout(/*1*/$arg0)
    {

    }
    /**
     * (int maxSize)
     * @return HttpClientRequest
     * methodSize : 1
     */
    public function setWriteQueueMaxSize(/*1*/$arg0)
    {

    }
    /**
     * ()
     * @return int
     * methodSize : 1
     */
    public function streamId(/*0*/)
    {

    }
    /**
     * ()
     * @return string
     * methodSize : 1
     */
    public function uri(/*0*/)
    {

    }
    /**
     * (Buffer data)
     * @return HttpClientRequest
     * (string chunk)
     * @return HttpClientRequest
     * (string chunk, string enc)
     * @return HttpClientRequest
     * methodSize : 3
     */
    public function write(/*2*/$arg0, $arg1)
    {

    }
    /**
     * (int type, int flags, Buffer payload)
     * @return HttpClientRequest
     * (HttpFrame frame)
     * @return HttpClientRequest
     * methodSize : 2
     */
    public function writeCustomFrame(/*3*/$arg0, $arg1, $arg2)
    {

    }
    /**
     * ()
     * @return bool
     * methodSize : 1
     */
    public function writeQueueFull(/*0*/)
    {

    }
}

File: vertx-lang-php-jphp/src/main/api/io/vertx/php/core/http/HttpClientRequest.java
package io.vertx.php.core.http;
import php.runtime.annotation.Reflection.Name;
import php.runtime.annotation.Reflection.Namespace;
import php.runtime.annotation.Reflection.Signature;
import io.vertx.lang.jphp.IWrapper;
import io.vertx.php.core.http.HttpClientResponse;
import io.vertx.php.core.buffer.Buffer;
import io.vertx.php.core.http.HttpFrame;
import io.vertx.php.core.streams.WriteStream;
import io.vertx.php.core.MultiMap;
import io.vertx.php.core.streams.ReadStream;
import io.vertx.php.core.http.HttpConnection;
@Name("HttpClientRequest")
@Namespace("io\\vertx\\php\\core\\http")
public interface HttpClientRequest<S extends io.vertx.core.http.HttpClientRequest> extends IWrapper<S>{
    
    @Signature
    default Memory writeQueueFull(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().writeQueueFull()
    }

    @Signature
    default Memory exceptionHandler(Environment __env__, Memory... args) {
        this.getWrappedObject().exceptionHandler(handler)
    }

    @Signature
    default Memory write(Environment __env__, Memory... args) {
        this.getWrappedObject().write(data)
    }

    @Signature
    default Memory setWriteQueueMaxSize(Environment __env__, Memory... args) {
        this.getWrappedObject().setWriteQueueMaxSize(maxSize)
    }

    @Signature
    default Memory drainHandler(Environment __env__, Memory... args) {
        this.getWrappedObject().drainHandler(handler)
    }

    @Signature
    default Memory handler(Environment __env__, Memory... args) {
        this.getWrappedObject().handler(handler)
    }
    
    @Signature
    default Memory pause(Environment __env__, Memory... args) {
        this.getWrappedObject().pause()
    }
    
    @Signature
    default Memory resume(Environment __env__, Memory... args) {
        this.getWrappedObject().resume()
    }

    @Signature
    default Memory endHandler(Environment __env__, Memory... args) {
        this.getWrappedObject().endHandler(endHandler)
    }

    @Signature
    default Memory setFollowRedirects(Environment __env__, Memory... args) {
        this.getWrappedObject().setFollowRedirects(followRedirects)
    }

    @Signature
    default Memory setChunked(Environment __env__, Memory... args) {
        this.getWrappedObject().setChunked(chunked)
    }
    
    @Signature
    default Memory isChunked(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().isChunked()
    }
    
    @Signature
    default Memory method(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().method()
    }
    
    @Signature
    default Memory getRawMethod(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().getRawMethod()
    }

    @Signature
    default Memory setRawMethod(Environment __env__, Memory... args) {
        this.getWrappedObject().setRawMethod(method)
    }
    
    @Signature
    default Memory absoluteURI(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().absoluteURI()
    }
    
    @Signature
    default Memory uri(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().uri()
    }
    
    @Signature
    default Memory path(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().path()
    }
    
    @Signature
    default Memory query(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().query()
    }

    @Signature
    default Memory setHost(Environment __env__, Memory... args) {
        this.getWrappedObject().setHost(host)
    }
    
    @Signature
    default Memory getHost(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().getHost()
    }
    
    @Signature
    default Memory headers(Environment __env__, Memory... args) {
        return Utils.convReturnVertxGen(__env__, MultiMapImpl::new, this.getWrappedObject().headers()
    }

    @Signature
    default Memory putHeader(Environment __env__, Memory... args) {
        this.getWrappedObject().putHeader(name, value)
    }

    @Signature
    default Memory continueHandler(Environment __env__, Memory... args) {
        this.getWrappedObject().continueHandler(handler)
    }
    
    @Signature
    default Memory sendHead(Environment __env__, Memory... args) {
        this.getWrappedObject().sendHead()
    }

    @Signature
    default Memory sendHead(Environment __env__, Memory... args) {
        this.getWrappedObject().sendHead()
    }
    
    @Signature
    default void end(Environment __env__, Memory... args) {
        this.getWrappedObject().end()
    }

    @Signature
    default void end(Environment __env__, Memory... args) {
        this.getWrappedObject().end()
    }

    @Signature
    default Memory setTimeout(Environment __env__, Memory... args) {
        this.getWrappedObject().setTimeout(timeoutMs)
    }

    @Signature
    default Memory pushHandler(Environment __env__, Memory... args) {
        this.getWrappedObject().pushHandler(handler)
    }
    
    @Signature
    default Memory reset(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().reset()
    }

    @Signature
    default Memory reset(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().reset()
    }
    
    @Signature
    default Memory connection(Environment __env__, Memory... args) {
        return Utils.convReturnVertxGen(__env__, HttpConnectionImpl::new, this.getWrappedObject().connection()
    }

    @Signature
    default Memory connectionHandler(Environment __env__, Memory... args) {
        this.getWrappedObject().connectionHandler(handler)
    }

    @Signature
    default Memory writeCustomFrame(Environment __env__, Memory... args) {
        this.getWrappedObject().writeCustomFrame(frame)
    }
    
    @Signature
    default Memory streamId(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().streamId()
    }

}
